src/sentry/static/sentry/app/components/events/relatedEvents/index.tsx

```diff
@@ -0,0 +1,167 @@
+
+import {CURRENT_LOCATION} from './types';
+
+enum EVENT_TYPE {
+  ERROR = 'error',
+  TRANSACTION = 'transaction',
+}
+
+type Props = {
+  relatedEvents: Array<TableDataRow>;
+  eventView: EventView;
+  organization: Organization;
+  currentLocation: CURRENT_LOCATION;
+  location: Location;
+};
+
+// List events that have the same tracing ID as the current Event
+const RelatedEvents = ({
+  currentLocation,
+  eventView,
+  relatedEvents,
+  organization,
+  location,
+}: Props) => {
+  const orgSlug = organization.slug;
+
+  const getTransactionLink = (transactionName: string, eventSlug: string) => {
+    return getTransactionDetailsUrl(
+      organization,
+      eventSlug,
+      transactionName,
+      location.query
+    );
+  };
+
+  const getEventTarget = (dataRow: TableDataRow & {type: EVENT_TYPE}) => {
+    if (currentLocation === CURRENT_LOCATION.DISCOVER) {
+      const eventSlug = generateEventSlug(dataRow);
+
+      return eventDetailsRouteWithEventView({
+        orgSlug,
+        eventSlug,
+        eventView,
+      });
+    }
+
+    if (dataRow.type === EVENT_TYPE.ERROR) {
+      return `/organizations/${orgSlug}/issues/${dataRow['issue.id']}/events/${dataRow.id}/`;
+    }
+
+    const eventSlug = generateEventSlug(dataRow);
+
+    return getTransactionLink(String(dataRow.title), eventSlug);
+  };
+
+  const renderEventId = (dataRow: TableDataRow & {type: EVENT_TYPE}) => (
+    <Tooltip title={t('View Event')}>
+      <StyledLink to={getEventTarget(dataRow)}>
+        {dataRow.id as React.ReactNode}
+      </StyledLink>
+    </Tooltip>
+  );
+
+  const renderIcon = (type: EVENT_TYPE) => {
+    if (type === EVENT_TYPE.ERROR) {
+      return <IconFire color="red400" />;
+    }
+    return <IconSpan color="pink400" />;
+  };
+
+  return (
+    <PanelTable headers={[t('Id'), t('Title'), t('Type'), t('Project'), t('Created')]}>
+      {relatedEvents.map((row, index) => {
+        const {id, title, project, timestamp} = row;
+
+        const eventType = row['event.type'] as EVENT_TYPE;
+        const isLast = index === relatedEvents.length - 1;
+
+        return (
+          <React.Fragment key={id}>
+            <StyledPanelItem isLast={isLast}>
+              {renderEventId({...row, type: eventType})}
+            </StyledPanelItem>
+            <StyledPanelItem isLast={isLast}>{title}</StyledPanelItem>
+            <StyledPanelItem isLast={isLast}>
+              <TypeWrapper>
+                {renderIcon(eventType)}
+                {capitalize(eventType)}
+              </TypeWrapper>
+            </StyledPanelItem>
+            <StyledPanelItem isLast={isLast}>
+              <StyledPlatformIcon platform={String(project)} size="16px" />
+              {project}
+            </StyledPanelItem>
+            <StyledPanelItem isLast={isLast}>
+              <TimeWrapper>
+                <IconClock size="16px" />
+                <StyledTimeSince date={timestamp} />
+                <div>{'-'}</div>
```

Comment:
```suggestion
                <div>{'\u2014'}</div>
```
Should this be an em dash?